ui/lib/core/addon/templates/components/info-table-row.hbs

```diff
@@ -5,24 +5,22 @@
   <div class="column is-flex">
     {{#if (has-block)}}
       {{yield}}
-    {{else if valueIsBoolean}}
+    {{else if (eq valueType "boolean")}}
       {{#if value}}
-        <Icon
-          aria-hidden="true"
-          class="icon-true"
-          @size="l"
-          @glyph="check-circle-outline"
-        /> Yes
+        <Icon aria-hidden="true" class="icon-true" @size="l" @glyph="check-circle-outline" /> Yes
       {{else}}
-        <Icon
-          aria-hidden="true"
-          class="icon-false"
-          @size="l"
-          @glyph="cancel-circle-outline"
-        /> No
+        <Icon aria-hidden="true" class="icon-false" @size="l" @glyph="cancel-circle-outline" /> No
       {{/if}}
     {{else}}
-      <code class="is-word-break has-text-black" data-test-row-value="{{label}}">{{value}}</code>
+      {{#if value}}
+        <code class="is-word-break has-text-black" data-test-row-value="{{label}}">{{value}}</code>
+      {{else}}
+        {{#if (eq valueType "array")}}
```

Comment:
Do we want to show `[]` or `{}` if there’s no value for it?

Reply:
It looks really weird if it’s just nothing imo. @joshuaogle what do you think?

Reply:
Maybe `No value assigned`? Or even `Default value` ? (but I’m not sure if there’s no value if it is the default value…). I think we should check for ttl types though - they’re currently showing a `{}` and I think we don’t want that.

What do you think about leaving these out for now and we can get them in after we talk through the various states - I agree that the blank is strange. Should we default config pages to hide items that are empty maybe?